### DISCUSSION TITLE
HAMRS POTA Template Clone

### DISCUSSION THREAD

**Discussion by mwfoutch** [magicbug/Cloudlog#3015] [Ideas / Feature Requests]
An efficient and popular logging software for Parks On The Air is HAMRS.  However, the developer of HAMRS hasn't updated it in a while and it is becoming less and less useful.  If Cloudlog included a selectable look that mimicked the HAMRS POTA template, it would be very useful and popular.  Much of the functionality is already there with Cloudlog and more.  The main thing missing for POTA with the Cloudlog main logging page is that a person needs to jump onto different tabs to change frequencies and enter park numbers.  Alternatively, if attempting to log POTA with a Cloudlog contest style, operator info doesn't seem to pull from QRZ when a call is entered (unless I'm doing something wrong). If Cloudlog had a look that could be selected as a template with just the relevant info for POTA on one page (like HAMRS) it would quickly become a favorite of POTA operators.  Is this something that could be accomplished?  If so, have a look at HAMRS and the request may be a bit more clear.  Contact me with any questions.  Thanks.  Sincerely,  AE4JC

**Comment by braydenwise** [comment]
I was literally about to write the same thing. 

For the devs: would it be possible to create user-defined/customizable QSO entry pages? 

For me, an ideal workflow would be to go QSO -> Live QSO -> POTA from the menu and have the following fields show on the first page:

Date/Time (as-is in current live QSO function)
Callsign
RST (S)
RST (R)
POTA Ref
Comments

Ideally, if we could create user-created/customizable QSO forms from within the current UI, that would rock. If there were a list of selectable fields (which you could either sort by using numerical order or drag-and-drop if we're getting fancy), operators could then customize what they want out of a given entry page.

This would have huge implications for mobile users especially, where it can be difficult to manoeuvre quickly between tabs in a QSO for logging park info. If I'm being honest, I'm decent with computers and UIs, but can't work fast enough to get the info in on my phone in a live situation, so I usually resort to paper logging and Simple FLE.

The other alternative would be to make a customizable area under Contests where we could add a field like POTA/SOTA/WWFF refs, etc, so we can quickly log those details.

Thoughts?

B.
VA7BKW

**Comment by magicbug** [comment]
I'll certainly take it on board i'm planning to improve the QSO entry page, making the whole thing customisable would not be an easy programming task and probably be quite a headache.
